README.md

```diff
@@ -37,16 +37,51 @@ npm install express-formidable --save
 touch views/signup.pug
 ```
 
+The sign up renders a Pug template for signing up. When the user makes a request from this form providing his username and password, we can create the user his account via the `tenancy` API. Later on, we can decide which type of wallet to create.
+
 ## Step 5 -- Clientele API, echo
 
+If you take a look at the `app.post('/login', ...)` function, you will see that we use the connection object for the `clientele` API. We test the connection object by requesting the API to sign our payload `foobar` which we send as data. The signgin request happens via the `echo` method.
+
+If we get a successful signed resonse from the API, we can be sure that the connection has been set up correctly.
```

Comment:
No signing. And it's not really a "connection" either. It's about verifying that the authentication for the API key works.